packages/components/psammead-play-button/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------------- | ----------- |
+| 1.1.0 | [PR#2610](https://github.com/bbc/psammead/issues/2610) Add `guidanceMessage` prop |
```

Comment:
can this be more descriptive like `Add psammead-media-player guidance message prop` atleast this shows where we are inferencing the prop from

Reply:
@RayNjeri thanks, but I think it's descriptive as it is, adding `psammead-media-player` ties it's usage to `psammead-media-player` but it's a standalone component that could be used in some other context